docs/source/install/index.rst

```diff
@@ -76,14 +76,13 @@ Download the source files from GitHub.
 
 .. code:: bash
 
-   git clone --recursive https://github.com/dmlc/dgl.git
+   git clone --recurse-submodules https://github.com/dmlc/dgl.git
```

Comment:
I don't think this change is needed since our CI build works well with `--recursive` option

Reply:
Does not work on may mac.

Reply:
Git-clone has deprecated `--recursive` in favor of the more verbose `--recurse-modules`. http://git.661346.n2.nabble.com/RFC-PATCH-clone-Deprecate-the-recursive-option-in-favor-of-recurse-submodules-td5702732.html